src/App.js

```diff
@@ -23,9 +27,14 @@ class App extends React.Component {
   render() {
     const { digitObj } = this.state;
     return (
-      <div className="App">
-        <Calculator updateState={this.updateState} digitObj={digitObj} />
-      </div>
+      <>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="calculator" element={<Calculator updateState={this.updateState} digitObj={digitObj} />} />
+          <Route path="quote" element={<Quote />} />
```

Comment:
Your app is not working as expected. I go the error below when I run your app on my machine. Kindly fix the error.

![image](https://user-images.githubusercontent.com/10332499/159833000-9c6cade3-5a93-4b8a-b78c-d0d93250ac7a.png)